applications/crossbar/src/modules/cb_search.erl

```diff
@@ -209,6 +315,45 @@ search(Context, Type, Query, Value) ->
 %%--------------------------------------------------------------------
 %% @private
 %% @doc
+%% Attempt to load a summarized listing of all instances of this
+%% resource.
+%% @end
+%%--------------------------------------------------------------------
+-spec multi_search(cb_context:context(), ne_binary(), wh_proplist()) -> cb_context:context().
+-spec multi_search(cb_context:context(), ne_binary(), wh_proplist(), wh_json:objects()) -> cb_context:context().
+multi_search(Context, Type, Props) ->
+    multi_search(Context, Type, Props , wh_json:new()).
+
+multi_search(Context, _Type, [], Acc) ->
+    Env = cb_context:resp_envelope(Context),
+    cb_context:set_resp_envelope(
+        cb_context:set_resp_data(Context, Acc)
+        ,wh_json:delete_keys([<<"start_key">>, <<"page_size">>], Env)
+    );
+multi_search(Context, Type, [{Query, Value}|More], Acc) ->
+    ViewName = <<?QUERY_TPL/binary, Query/binary>>,
+    ViewOptions = [
+        {'startkey', get_start_key(Context, Type, Value)}
```

Comment:
I don't think we drop the first tuple down a line; should either be put back up a line or drop the `[` down to this line.